assets/js/components/notifications/index.js

```diff
@@ -150,19 +150,33 @@ const Notifications = ({methods, constants, ...props}) => {
         );
     }
 
-    return (
-        <div className={methods.classnames('newfold-notifications-wrapper')}>
-            {activeNotifications.map(notification => (
+    if ('bluehost-app-nav' === constants.context && activeNotifications.length > 0) {
```

Comment:
Should this be `container()->plugin()->id . '-app-nav'`?

Reply:
Good catch! fixed.